src/main/resources/hudson/plugins/gradle/Gradle/help-rootBuildScriptDir.html

```diff
@@ -1,5 +1,5 @@
 <div>
     If your workspace has the top-level build.gradle in somewhere other than the module root directory, specify the path
     (relative to the module root) here,
-    such as ${workspace}/parent/ instead of just ${workspace}. If left empty, defaults to build.gradle
+    as /parent/ instead of just ${WORKSPACE} where ${WORKSPACE}/parent is the absolute. If left empty, defaults to build.gradle in ${WORKSPACE} directory.
```

Comment:
```suggestion
    as parent where ${WORKSPACE}/parent is the absolute. If left empty, defaults to build.gradle in ${WORKSPACE} directory.
```

`/parent` will reference to the absolute path. It should be `parent` instead.